test/correctness/recursive_box_filters.cpp

```diff
@@ -26,6 +26,10 @@ int main(int argc, char **argv) {
     // have to pass 'true' to the atomic call to tell it to skip the check.
     h.update(2).atomic(true).vectorize(r, 16);
 
+    // Silence warnings
```

Comment:
Nit: I think this comment should be "These stages don't need scheduling"

Reply:
Done